src/analysis/souffle/examples/sql_examples/query6.dl

```diff
@@ -0,0 +1,29 @@
+#include "taint.dl"
+#include "sql_aux.dl"
+#include "policy.dl"
+
+isAccessPath("top.exprResult").
```

Comment:
Add sql query being modeled as a comment here and elsewhere to understand the facts?

Reply:
sounds good, will do.